pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -111,4 +111,17 @@ type CiliumBGPVirtualRouter struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:MinItems=1
 	Neighbors []CiliumBGPNeighbor `json:"neighbors"`
+	// InjectedRoutes is a list of BGP Routes to be announced for this virtual router
+	//
+	// +kubebuilder:validation:Optional
+	InjectedRoutes CiliumBGPInjectedRoute `json:"injectedRoutes"`
```

Comment:
as this one is optional, any reason for not using a pointer instead of a value here? Also omitempty will be useful as well.

Reply:
Slipped my mind, updated

Additionally checked in the api crd I'd apparently forgotten